src/models/session.js

```diff
@@ -1,8 +1,10 @@
-import api from '../services/api'
+import api from 'services/api'
+import { getUserEOSAccout } from 'services/scatter'
 
 const initialState = {
   accessToken: null,
-  user: null
+  user: null,
+  scatter: null
```

Comment:
Small thing, neither `user` nor `scatter` are currently being used, can we replace them with `account`?

Reply:
got catch

Reply:
good catch